data/quarg/quarg outfits.txt

```diff
@@ -16,8 +16,8 @@ outfit "Nanotech Battery"
 	category "Power"
 	cost 3000000
 	thumbnail "outfit/quarg nanotech battery"
-	"mass" 50
-	"outfit space" -50
+	"mass" 25
+	"outfit space" -25
 	"energy capacity" 60000
 	description "Human engineers who first came into contact with the Quarg were astonished by the sheer efficiency of their technology. The energy density of Quarg batteries is unparalleled, storing nearly a third more energy than the best human batteries while being less than a third the size."
```

Comment:
```suggestion
	"mass" 50
	"outfit space" -50
	"energy capacity" 60000
	description "Human engineers who first came into contact with the Quarg were astonished by the sheer efficiency of their technology. The energy density of Quarg batteries is unparalleled, storing nearly a third more energy than the best human batteries while being nearly a third smaller."
```

I would update the description to match the stats rather than upgrade the stats to match the description. 

Just halving the size as you've done here will massively increase the battery's overall effectiveness, measured in terms of how much energy capacity it provides versus how much space it takes up.

Previously, the second most space-efficient battery, the Crystal Capacitor, provided 1,133.33 Capacity/Space, with a downside of drawing energy. 
The best battery was the Nanotech Battery at 1200 C/S. 

With this change, the Nanotech Battery would have 2400 C/S, more than double the second-best battery in the game, which is far too much.

<details> 
<summary>Chart</summary> 

![image](https://github.com/user-attachments/assets/7587b118-15f2-4b0a-897d-96f35feb885c)
</details>

Reply:
That's what I was wondering, but if you look at the comment in the issue Hurleveur thought efficiency should be increased as opposed to the description tweaked.
However, I do think that game-design-wise, it's better to change the description.

Reply:
> Except for the whole issue is that the battery _is no longer_ about a third the size of human batteries, but is rather now 5/8ths the size.

Yes, it is no longer a third the size of the largest human battery, hence why I've changed it to say nearly a third smaller rather than a third the size. 

One third the size would be ~26
One third smaller is ~53

The nanotech battery is 50t.

Reply:
Ah yes, right.
```suggestion
	"mass" 25
	"outfit space" -25
	"energy capacity" 60000
	description "Human engineers who first came into contact with the Quarg were astonished by the sheer efficiency of their technology. The energy density of Quarg batteries is unparalleled, storing nearly a third more energy than the best human batteries while being over a third smaller."
```